packages/nimble-components/docs/component-status.stories.ts

```diff
@@ -253,6 +253,15 @@ const components = [
         angularStatus: '✅',
         blazorStatus: '✅',
     },
```

Comment:
Something strange is going on with beachball. This PR doesn't have any change files but the beachball status check in the builds passed. It claims that change files already exist for the packages being modified. 

I'll look into a bit but leaving this comment to prevent merging until we figure it out.

Reply:
Oh, I see, it's probably because the PR is currently merging into `rich-text-editor-viewer-spec` rather than `main`. Once the spec PR is completed and this is updated and retargeted to `main`, you'll need to run `npm run change` to create a change file.

Reply:
I'm assuming that's unexpected and this should target main? The branch target can be updated for the PR by clicking edit at the top.

Reply:
This branch is taken from the `rich-text-editor-viewer-spec` branch (spec PR) which already has the change file and it is still open (not merged to `main`) so I couldn't add the change file to this PR. Once the spec PR is completed, I will retarget the branch of this PR to `main` and will add the change file. Until then, we will have this comment open.

Reply:
As the spec PR branch is merged to `main`, I have retargeted this branch to `main` and added the change file.